x-pack/test_serverless/README.md

```diff
@@ -64,9 +64,9 @@ following namesapces:
 ### Adding Serverless Tests
 
 As outlined above, serverless tests are separated from stateful tests (except
-the reuse helper methods), which includes a separate base configuration. All
-tests that should run in an serverless environment have to be added here in
-`x-pack/test_serverless`.
+the reuse of helper methods), which includes a new base configuration. All
+tests that should run in a serverless environment have to be added to, in the 
+`x-pack/test_serverless` directory.
```

Comment:
The change of the end of the last sentence doesn't make sense to me:
```
All tests that should run in a serverless environment have to be added to, in the `x-pack/test_serverless` directory.
```
If the reason for that change was to get rid of the `here`, I'd suggest something like this:
```
All tests that should run in a serverless environment have to be added to the `x-pack/test_serverless` directory.
```

Reply:
@pheyos. the change is only in grammar, not in any meaning. 

Before, the sentence _implied_ that `x-pack/test_serverless` is a "thing". The "thing" could be a file/folder/different repo etc and doesn't specify how to add tests (in a file? in a domain folder?). 

The change makes it more specific: add a new test as a new file in the folder `x-pack/test_serverless`.

It's minor and a nit from me. I can revert that if you want.

Reply:
@TinaHeiligers thatnks for providing additional context. I don't have a strong opinion on the wordsmithing here, so you can go ahead and merge your suggestions.